lib/phoenix_html/form.ex

```diff
@@ -598,6 +648,16 @@ defmodule Phoenix.HTML.Form do
     generic_input(:hidden, form, field, opts)
   end
 
+  @doc """
+  Generates hidden inputs for the given form.
+  """
+  @spec hidden_inputs_for(t) :: list(Phoenix.HTML.safe())
+  def hidden_inputs_for(form) do
+    for {k, v} <- form.hidden do
+      hidden_input(form, k, value: v)
+    end
```

Comment:
Shall we also use this function [here](https://github.com/phoenixframework/phoenix_html/pull/292/files#diff-149975a43f9e90f073bd077072ba7d50R471)?

Reply:
is that what you mean here?

https://github.com/phoenixframework/phoenix_html/pull/292/files#diff-3a77d7e0ca67e4df7840545f22fa4218R444

Reply:
Sorry, it seems I sent the wrong link. I wanted to propose to use `hidden_inputs_for` here: https://github.com/phoenixframework/phoenix_html/blob/master/lib/phoenix_html/form.ex#L421-L422

Reply:
Done!